data-config.json

```diff
@@ -22,6 +22,32 @@
         "content-exporter": false,
         "model-adaptables": ["request"],
         "generic-javadoc": false,
+        "properties-tabs" : [
+            {
+                "id": "tab-1",
+                "label": "Tab 1",
+                "fields": ["textfieldTest", "checkTest", "pathfieldTest", "textareaTest", "dateTest", "selectTest", "radioTest", "hiddenTest", "numberfieldTest"]
+            },
+            {
+                "id": "tab-2",
+                "label": "Tab 2",
+                "fields": ["imageTest","colors","links"]
```

Comment:
nitpick, but let's add spaces after the commas, for consistency with line 29 above

Reply:
Done